{"name": "test_login_valid", "status": "broken", "statusDetails": {"message": "AttributeError: 'ConfigReader' object has no attribute 'get_user_credentials'", "trace": "self = <tests.test_login.TestLoginWeb object at 0x10754dbd0>\n\n    @allure.story(\"Valid Login\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_login_valid(self):\n        login_page = LoginPage(self.driver)\n>       user_creds = self.config.get_user_credentials(\"standard_user\")\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       AttributeError: 'ConfigReader' object has no attribute 'get_user_credentials'\n\ntests/test_login.py:13: AttributeError"}, "attachments": [{"name": "Lá»—i: test_login_valid_failure.png", "source": "41d14978-bc08-4bc1-9861-229244db1762-attachment.png", "type": "image/png"}], "start": 1753588150887, "stop": 1753588150887, "uuid": "28e3570d-6b14-4737-a180-55deffb8a01a", "historyId": "fa1e8a356caf3e31ce886507ffb95456", "testCaseId": "fa1e8a356caf3e31ce886507ffb95456", "fullName": "tests.test_login.TestLoginWeb#test_login_valid", "labels": [{"name": "story", "value": "Valid Login"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLoginWeb"}, {"name": "host", "value": "DUCs-MacBook-Air.local"}, {"name": "thread", "value": "80702-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}